src/pages/docs/getting-started/laravel.mdx

```diff
@@ -0,0 +1,398 @@
+---
+title: "Getting started: Pub/Sub in Laravel"
+meta_description: "A getting started guide for Ably Pub/Sub Laravel 12 that steps through some of the key features using Laravel."
+meta_keywords: "Pub/Sub Laravel 12, Laravel PubSub, Ably Laravel SDK, publish Laravel, Ably Pub/Sub guide, Laravel realtime communication, Ably tutorial Laravel, Laravel message history, presence API Laravel, Ably Pub/Sub example, realtime Pub/Sub Laravel, publish message Laravel, Ably CLI Pub/Sub"
+languages:
+  - PHP
+---
+
+This guide will get you started with Ably Pub/Sub in Laravel 12.
+
+It will take you through the following steps:
+
+* Create a client and establish a connection to Ably with the Laravel Broadcaster and Echo.
```

Comment:
Should we provide some context on the broadcaster (should this be Ably Laravel Broadcaster?) and that we use a fork of echo? WDYT?